test/integration/runtime/current/ratelimit/config/another_legacy.yaml

```diff
@@ -6,16 +6,6 @@ descriptors:
       requests_per_unit: 20
 
   - key: key3
-    rate_limit:
-      unit: hour
-      requests_per_unit: 10
-
-  - key: key2_local
-    rate_limit:
-      unit: minute
-      requests_per_unit: 20
-
-  - key: key3_local
```

Comment:
Sorry why are these removed here and below?

Reply:
Because we just made the change to undo the local cache test in TestBasicConfigLegacy (https://github.com/lyft/ratelimit/pull/114#issuecomment-574914116), i removed the related data as well.

Reply:
OK got it, thanks.